### PR TITLE
IEN-616 - Fixed withdrawn users not being shown in report 3

### PR DESCRIPTION
**Pull request by dbayly-freshworks**

### CODE REVIEW

File: packages/common/src/enum/milestone-status.ts [hunk 1]
```diff
@@ -121,3 +121,48 @@ export enum StatusId {
   SENT_EMPLOYER_DOCUMENTS_TO_HMBC = '1651cad1-1e56-4c79-92ce-f548ad9ec52c',
   WITHDREW_FROM_COMPETITION = '3fd4f2b0-5151-d7c8-6bbc-3a0601b5e1b0',
 }
+
+export enum STATUS_REASONS {
+  COST_OF_PROCESS = 'Cost of the IEN process',
+  TIMING = 'Timing',
+  WITHDREW_FROM_PROGRAM = 'Withdrew from IEN program',
+  PURSUING_LICENSE_IN_ANOTHER_PROVINCE = 'Pursuing licensure in another province',
+  ACCPTED_JOB_OUTSIDE_BC = 'Accepted another job outside of BC',
```
Comment (coryatwater-FW) [inline packages/common/src/enum/milestone-status.ts:130]: spelling

File: packages/common/src/enum/milestone-status.ts [hunk 2]
```diff
@@ -121,3 +121,48 @@ export enum StatusId {
...
+  ACCEPTED_JOB_IN_BC = 'Accepted another job in BC',
+  ACCPTED_NURSING_JOB_OUTSIDE_BC = 'Accepted another nursing job (public or private) outside of BC',
```
Comment (coryatwater-FW) [inline packages/common/src/enum/milestone-status.ts:132]: spelling on lines 130 and 132

File: packages/common/src/enum/milestone-status.ts [hunk 3]
```diff
@@ -121,3 +121,48 @@ export enum StatusId {
...
+  IMMIGRATION_ISSUE = 'Immigration concerns/issues',
+}
+export enum CandidateWithdrawReasonId {
+  COST_OF_PROCESS = 'FF4B82C9-1E4D-4561-84AA-08EB2DD9D37E',
+  TIMING = 'BFFAFBFD-EDFD-4F62-B72B-25E33BB55992',
+  WITHDREW_FROM_PROGRAM = 'A1E0E2C1-E272-468A-8F7A-28686B750CAE',
+  PURSUING_LICENSE_IN_ANOTHER_PROVINCE = 'E0930E51-630A-474F-BAE1-31E52A552CBB',
+  ACCPTED_JOB_OUTSIDE_BC = '74905D23-EF3D-453C-A2F2-45E4E2798A0B',
+  ACCEPTED_JOB_IN_BC = '1886B20E-17FF-41E6-8915-ECF6E03900DB',
+  ACCPTED_NURSING_JOB_OUTSIDE_BC = '3F84D93D-C122-4895-8D49-4D975F0D0005',
```
Comment (coryatwater-FW) [inline packages/common/src/enum/milestone-status.ts:154]: spelling on lines 152 and 154

File: packages/common/src/enum/milestone-status.ts [hunk 4]
```diff
@@ -121,3 +121,48 @@ export enum StatusId {
...
+  OTHER = 'E2958403-CB56-4396-B903-654C1FE787CC',
+  COMPENSATION = '583C72E3-6D58-4E0B-ADC1-7F5E89A62CD5',
+  REVOKED_JOB_SEARCH = 'C09A8B50-7FE5-4EED-84E5-810CCB17F5D4',
+  UNRESPONSIVE = 'ED497BF5-4E1D-46A0-852B-921628125839',
+  LIMITED_ACCOMODATION = 'C9C6BD85-35AC-443D-91D5-C488594622AB',
+  JOB_SUITABILITY = 'FD756221-0F02-4484-9F8F-DB78E94E78EB',
+  PERSONAL_REASONS = '49873A72-AB2D-4349-AD4D-EE49CEB8288F',
+  CANNOT_MEET_PROGRAM_CRITERIA = '1824580D-ED74-4E37-BA1C-F15DBB3878D2',
+  IMMIGRATION_ISSUE = '031CBA24-8B47-4AD8-A932-FA2FA858B25B',
+}
```
Comment (coryatwater-FW) [inline packages/common/src/enum/milestone-status.ts:168]: are the queries case-sensitive? my db has them all in lower case

Comment (dbayly-freshworks) [reply packages/common/src/enum/milestone-status.ts:168]: They don't seem to be, but I can make these lower case for consistency

File: packages/common/src/enum/milestone-status.ts (hunk 2 above)
Comment (dbayly-freshworks) [reply packages/common/src/enum/milestone-status.ts:132]: Thanks!

**Pull request closed without merging**